build/networking_ebpf/Dockerfile.ubuntu

```diff
@@ -69,6 +69,7 @@ RUN apt-get install -y apt-utils \
 
 RUN pip install --upgrade pip
 RUN pip install grpcio \
+            grpcio-tools \
```

Comment:
require to fix atleast on ubuntu 18.04
: program not found or is not executable
--grpc_out: protoc-gen-grpc: Plugin failed with status code 1.

Reply:
Nice catch!